architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -34,6 +37,44 @@ you are responsible for updating it to reflect requirements of newer versions of
 the {product-title} master.
 ====
 
+=== General Admission Rules
+Starting in 1.3, OpenShift uses a single admission chain for Kubernetes and OpenShift
+resources.  This changed from 1.2 and before where we had separate admission chains.
+This means that the top level `admissionConfig.pluginConfig` element can now contain
+admission plugin configuration which used to be contained in `kubernetesMasterConfig.admissionConfig.pluginConfig`.
+
+The `kubernetesMasterConfig.admissionConfig.pluginConfig` should be moved and merged into
+`admissionConfig.pluginConfig`.  
+
+Also starting in 1.3, all the supported admission plugins are ordered in the single chain
+for you.  You should no longer have to set `admissionConfig.pluginOrderOverride` or 
```

Comment:
should this be stronger (e.g. "You should no longer set", and "Instead, you should enable")?